linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -43,6 +43,21 @@ graph TD
     B--> C(gaps occured)-->E(memmove)-->F(resize done)
 
 ```
+### Threadpool timer
```

Comment:
>Threadpool timer [](http://example.com/codeflow?start=4&length=16)

One thing that may be helpful is to mention the bug that we found with the delete.

Reply:
Added some text to describe the problem.